python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1446,6 +1446,186 @@ keys:
+                    port_id:
+                      type: int
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 4094
+                      description: Tap port ID tag.
+                    inner_port_id:
+                      description: Inner tap port ID tag.
+                      type: int
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 4094
+                mpls_pop_all:
+                  type: bool
+                  description: Pop all MPLS labels.
+                native_vlan:
+                  type: int
+                  convert_types:
+                    - str
+                  min: 1
+                  max: 4094
+                  description: Native VLAN ID when interface is in tap mode.
+                truncation:
+                  type: dict
+                  keys:
+                    enabled:
+                      type: bool
+                      required: true
+                    size:
+                      description: Ingress packet truncation size in bytes.
+                      type: int
+                      convert_types:
+                        - str
+                      min: 100
+                      max: 9236
+                mac_address:
+                  type: dict
+                  keys:
+                    src:
+                      type: str
+                      pattern: "^([0-9a-f]{2}:){5}[0-9a-f]{2}$"
+                      description: MAC address for the source.
+                    dest:
+                      type: str
+                      required: true
+                      pattern: "^([0-9a-f]{2}:){5}[0-9a-f]{2}$"
+                      description: MAC address for the destination.
+                encapsulation:
+                  type: dict
+                  keys:
+                    vxlan_strip:
+                      type: bool
+                      description: |-
+                        Strip VXLAN encapsulation header.
+                        `encapsulation.vxlan_strip` and `mpls_pop_all` are mutually exclusive.
+                        `mpls_pop_all` takes precedence.
+                    gre:
+                      type: list
+                      items:
+                        type: dict
+                        keys:
+                          strip:
+                            type: bool
+                            required: true
+                            description: Strip GRE encapsulation header.
+                          destination:
+                            description: Destination IP address of tunnel packets.
+                            type: str
+                          source:
+                            description: Source IP address of tunnel packets. Applied only when destination is defined.
+                            type: str
+                          protocol:
```

Comment:
It also has `feature` 
eg. `switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0xad9c feature header length 3 strip`
We can have a discussion on it.

Reply:
```
   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x1 feature header length 2 strip
   switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0x0 strip
   switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0x1 feature header length 4 strip
   switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0x101 feature header length 4 strip
   switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0xad9c feature header length 3 strip
   ```

Reply:
As discussed, it does not reflect on the config if the protocol is `0x000` otherwise it works.
I will update the schema.
Thanks.

Reply:
Fixed!